package.json

```diff
@@ -106,11 +104,14 @@
     "collectCoverageFrom": [
       "src/**/*.{js,jsx}"
     ],
+    "testPathIgnorePatterns": [
+      "<rootDir>/src/client/actions/__test__/utils.js" 
```

Comment:
Look like wild card is not implemented. Meaning for every helpers , I need to add a hard path to get them ignored. I didn't find any solution.